test/opf/ed.jl

```diff
@@ -0,0 +1,85 @@
+function test_opf_pm(::Type{OPFGenerator.EconomicDispatch}, data::Dict)
+    OPF = OPFGenerator.EconomicDispatch
+
+    data["basic_network"] || error("Input data must be in basic format to test")
+    G = length(data["gen"])
+    E = length(data["branch"])
+
+    # Build and solve OPF with OPFGenerator
+    solver = OPT_SOLVERS[OPF]
+    opf = OPFGenerator.build_opf(OPF, data, solver) # NOTE: this does not consider soft versions
+    OPFGenerator.solve!(opf)
+    res = OPFGenerator.extract_result(opf)
+
+    # Solve OPF with PowerModels
+    # must be after OPFGenerator since it modifies `data`
+    solver = OPT_SOLVERS[OPF]
+    res_pm = PM.solve_opf_ptdf_branch_power_cuts!(data, solver)
+
+    # Check that the right problem was indeed solved
+    @test res["opf_model"] == string(OPF)
+    @test res["termination_status"] ∈ [LOCALLY_SOLVED, OPTIMAL]
+    @test res["primal_status"] == FEASIBLE_POINT
+    @test res["dual_status"] == FEASIBLE_POINT
+    # Check objective value against PowerModels
+    @test isapprox(res["objective"], res_pm["objective"], atol=1e-6, rtol=1e-6)
+    @test res["ptdf_iterations"] == res_pm["iterations"]
```

Comment:
I don't think we should test this. If we change any algorithm setting, we might get different iteration count and fail tests, although the code would do what it's supposed to do.